### ISSUE TITLE
Zmienny styl tekstu

### ISSUE THREAD

**Issue by Cideu** [CodoDodo/Zakop#65]
Błąd występuje po rozwinięciu długiego wpisu.
https://streamable.com/5hxzc

**Comment by Ciema** [comment]
@Cideu wyglada troche jak cytowany tekst.

**Comment by CodoDodo** [comment]
@Ciema Bo to jest cytowany tekst ;)
@Cideu To nie jest tak naprawdę błąd, po prostu ten krótszy tekst jest pomniejszony o cytowanie i wygląda to tak jak wygląda. Ciężko będzie to naprawić

**Comment by Cideu** [comment]
@CodoDodo @Cideu  Tak, to cytat i sporo wpisów pojawia się w takim stylu. Może warto zmienić sposób zawijania tekstu tak, aby nie wpływało na jego format?
